### DISCUSSION TITLE
How can update check be disabled for macOS Homebrew (packaged) installs?

### DISCUSSION THREAD

**Discussion by luckman212** [netdata/netdata#17845] [Q&A]
Currently there's a big yellow ⚠️ mark on my local dashboard due to netdata insisting that I have an outdated version.

I've run `brew update && brew upgrade netdata` already and this is the latest version available (but of course is a bit behind the main branch).

How can I disable this check to avoid the warning which I can do nothing about? I want to stick with the Homebrew install because it simplifies installation and management.

![screenshot_8rShd31w](https://github.com/netdata/netdata/assets/1992842/7d1f9d18-f084-41af-bb00-f92ce86c1126)

**Comment by luckman212** [comment]
Also, I have read https://github.com/netdata/netdata/discussions/13073 and there is no `netdata-updater.sh` file anywhere on my disk.

**Comment by ilyam8** [comment]
Hey, @luckman212

> How can I disable this check to avoid the warning which I can do nothing about?

You can't.

> I've run brew update && brew upgrade netdata already and this is the latest version available

We do not package homebrew Netdata. I see that the latest version of the homebrew package is v1.44.3 and the latest stable version of Netdata is v1.45.6.

We changed our build system to CMake in version 1.45.0, and it seems that maintainers of the homebrew Netdata package can't get it to work for them (e.g. https://github.com/Homebrew/homebrew-core/pull/173783).

---

cc @Ferroin @vkalintiris I think we need either to help guys (if they want) to solve problems with Netdata v1.45.0+ or remove "Install Netdata via Homebrew" from our docs and probably even discourage people from doing it.

**Reply by luckman212** [reply]
Ah, thanks for clarifying @ilyam8. I'll probably just switch to `netdata-kickstart.sh` method if that works. Homebrew was appealing because it's so simple on macOS but TBH the one-liner kickstart install isn't too much harder! I subscribed to that issue you linked and will watch for changes there.

**Comment by luckman212** [comment]
It looks like https://github.com/Homebrew/homebrew-core/pull/173783 is marked "Closed" so I'm not sure where this goes from here.

**Reply by ilyam8** [reply]
Yes, nothing is going on there. That is why I said

> We changed our build system to CMake in version 1.45.0, and it seems that maintainers of the homebrew Netdata package can't get it to work for them

and

> cc @Ferroin @vkalintiris I think we need either to help guys (if they want) to solve problems with Netdata v1.45.0+ or remove "Install Netdata via Homebrew" from our docs and probably even discourage people from doing it.

**Reply by luckman212** [reply]
Sorry for the extra messaging. Yes, I did try to switch to the kickstart script on macOS but it also failed for me on 2 systems so I do hope the Homebrew crew can figure out how to get it building again.

I will try to debug the manual install problems separately and open a different issue if needed.
